godump_test.go

```diff
@@ -891,6 +869,27 @@ func TestDumpWithCustomWriter(t *testing.T) {
 	}
 }
 
+func TestDumpWithCustomSkipStackFrames(t *testing.T) {
+	// caller stack frames are
+	//	1	godump.go           github.com/goforj/godump.findFirstNonInternalFrame			skip by initialCallerSkip
+	//	2	godump.go           github.com/goforj/godump.printDumpHeader					skip by initialCallerSkip
+	//	3	godump.go           github.com/goforj/godump.(*Dumper).DumpStr					skip by fail names contain godump.go
+	//	4	godump_test.go      github.com/goforj/godump.TestDumpWithCustomSkipStackFrames
+	//	5	testing.go          testing.tRunner
+	out := NewDumper().DumpStr("test")
+	assert.Contains(t, out, "godump_test.go")
```

Comment:
Here you may check the output contains the name of the test, you can hardcode it here

assert.Contains(t, out, "TestDumpWithCustomSkipStackFrames")